internal/settings.go

```diff
@@ -161,3 +162,16 @@ func (ds *DialSettings) Validate() error {
 	}
 	return nil
 }
+
+// UniverseDomain returns the default service domain for a given Cloud universe.
+// The default value is "googleapis.com".
+func (ds *DialSettings) GetUniverseDomain() string {
+	if ds.UniverseDomain == "" {
+		return universeDomainDefault
+	}
+	return ds.UniverseDomain
+}
+
+func (ds *DialSettings) UniverseDomainNotGDU() bool {
```

Comment:
nit: consider starting with `Is`. And do we want the negation in text or code.

Reply:
Done, thanks!